frappe/utils/response.py

```diff
@@ -242,16 +246,23 @@ def json_handler(obj):
 	elif callable(obj):
 		return repr(obj)
 
-	elif isinstance(obj, uuid.UUID):
+	elif isinstance(obj, Path):
 		return str(obj)
 
-	elif isinstance(obj, Path):
+	# orjson does this already
+	# but json_handler needs to be compatible with built-in json module also
+	elif isinstance(obj, UUID):
 		return str(obj)
 
 	else:
 		raise TypeError(f"""Object of type {type(obj)} with value of {obj!r} is not JSON serializable""")
 
 
+dump_response = functools.partial(
+	orjson.dumps, default=json_handler, option=orjson.OPT_PASSTHROUGH_DATETIME | orjson.OPT_NON_STR_KEYS
```

Comment:
Can you add an explanation for these options.

Reply:
how about a util called `orjson_dumps` with these options are default? I will add an explanation of the options there.

will add this util in `safe_exec` as well, and then use it for `frappe.boot`.